__tests__/e2e/docker-compose.yml

```diff
@@ -40,7 +40,7 @@ services:
     networks:
       - elastic
   synthetic:
-    image: docker.elastic.co/experimental/synthetics:${STACK_VERSION}-synthetics
+    image: docker.elastic.co/beats/heartbeat:${STACK_VERSION}
```

Comment:
We don't need to support previous versions? Only 7.11/master/7.x? If so ok, otherwise maybe the docker-compse should support the previous behaviour?

Reply:
synthetic is experimental (https://www.elastic.co/what-is/synthetic-monitoring) the first version of heartbeat with synthetic is 7.11, so we do not need to support older versions